packages/documentation/copy/en/reference/Type Compatibility.md

```diff
@@ -288,3 +288,30 @@ These differ only in that assignment extends subtype compatibility with rules to
 For practical purposes, type compatibility is dictated by assignment compatibility, even in the cases of the `implements` and `extends` clauses.
+
+## Any, unknown, object, void, undefined, null, and never
+
+The following table summarizes assignability between some abstract types.
+Rows indicate what each is assignable to, columns indicate what is assignable to them.
+A ":white_check_mark:" indicates a combination that is compatible only when [`--strictNullChecks`](/tsconfig#strictNullChecks) is off.
+
+|             |        any         |      unknown       |       object       |        void        |     undefined      |        null        | never |
+| ----------- | :----------------: | :----------------: | :----------------: | :----------------: | :----------------: | :----------------: | :---: |
+| any →       |                    | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |  :x:  |
+| unknown →   | :heavy_check_mark: |                    |        :x:         |        :x:         |        :x:         |        :x:         |  :x:  |
+| object →    | :heavy_check_mark: | :heavy_check_mark: |                    |        :x:         |        :x:         |        :x:         |  :x:  |
+| void →      | :heavy_check_mark: | :heavy_check_mark: |        :x:         |                    |        :x:         |        :x:         |  :x:  |
+| undefined → | :heavy_check_mark: | :heavy_check_mark: | :white_check_mark: | :heavy_check_mark: |                    | :white_check_mark: |  :x:  |
+| null →      | :heavy_check_mark: | :heavy_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |                    |  :x:  |
+| never →     | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |       |
+
+Reiterating [Basic Types](/handbook/basic-types.html):
+- Everything is assignable to itself.
+- `any` and `unknown` are the same in terms of what is assignable to them, different in that `unknown` is not assignable to anything except `any`.
+- `unknown` and `never` are like inverses of each other.
+  Everything is assignable to `unknown`, `never` is assignable to everything.
+  Nothing is assignable to `never`, `unknown` is not assignable to anything (except `any`).
+- `void` is not assignable to or from anything, with the following exceptions: `any`, `unknown`, `never`, `undefined`, and `null` (if `--strictNullChecks` is off, see table for details).
+- When `--strictNullChecks` is off, `null` and `undefined` are similar to `never`: assignable to most types, most types are not assignable to them.
+  They are assignable to each other.
+- When `--strictNullChecks` is on, they behave more like `void`: not assignable to or from anything, except for `any`, `unknown`, `never`, and `void` (`undefined` is always assignable to `void`).
```

Comment:
```suggestion
- When `--strictNullChecks` is on, `null` and `undefined` behave more like `void`: not assignable to or from anything, except for `any`, `unknown`, `never`, and `void` (`undefined` is always assignable to `void`).
```

No need to keep what we are referring to in your head, IMO (in a re-read I had to go back to the previous line to remember what we're talking about)